op-deployer/pkg/deployer/standard/standard.go

```diff
@@ -43,6 +43,7 @@ const (
 	ContractsV400Tag        = "op-contracts/v4.0.0-rc.7"
 	ContractsV410Tag        = "op-contracts/v4.1.0"
 	ContractsV500Tag        = "op-contracts/v5.0.0"
+	ContractsV600Tag        = "op-contracts/v6.0.0"
 	CurrentTag              = ContractsV500Tag
```

Comment:
Shouldn't `CurrentTag` become `op-contracts/v6.0.0` now? Will need to update the superchain-registry import as well to get that to work if it doesn't already include the v6.0.0 standard contract addresses